src/components/Tables/AppletsTable/Table/AppletItem/AppletItem.tsx

```diff
@@ -1,52 +1,93 @@
+import { useState } from 'react';
 import { useNavigate } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
 import { TableCell, TableRow } from '@mui/material';
+import { Box } from '@mui/system';
 
 import { useTimeAgo } from 'hooks';
 import { useAppDispatch } from 'redux/store';
-import { FolderApplet, folders } from 'redux/modules';
-import { StyledBodyMedium } from 'styles/styledComponents/Typography';
+import { account, FolderApplet, folders } from 'redux/modules';
+import { StyledBodyMedium, StyledHeadline } from 'styles/styledComponents/Typography';
+import { StyledModalBtn, StyledModalText, StyledModalWrapper } from 'styles/styledComponents/Modal';
 import { Actions } from 'components/Actions';
-import { appletPages } from 'utils/constants';
 import { Pin } from 'components/Pin';
+import { BasicPopUp } from 'components/Popups/BasicPopUp';
+import { appletPages } from 'utils/constants';
+import { useAsync } from 'hooks/useAsync';
+import { deleteAppletApi } from 'api';
 
 import { AppletImage } from '../AppletImage';
 import { StyledAppletName, StyledPinContainer } from './AppletItem.styles';
-import { actions } from './AppletItem.const';
+import { actionsRender } from './AppletItem.const';
 
 export const AppletItem = ({ item }: { item: FolderApplet }) => {
+  const { t } = useTranslation('app');
   const dispatch = useAppDispatch();
   const navigate = useNavigate();
   const timeAgo = useTimeAgo();
+  const accountData = account.useData();
+  const [deleteModalShowed, setDeleteModalShowed] = useState(false);
```

Comment:
I think it's more convenient and grammatically correct to use Active instead Showed